testsuite/testcases/src/lib.rs

```diff
@@ -212,6 +212,22 @@ impl dyn NetworkLoadTest {
             .build()
             .map_err(|err| anyhow!("Failed to start runtime for transaction emitter. {}", err))?;
 
+        let clients = ctx
+            .swarm()
+            .get_clients_for_peers(&nodes_to_send_load_to, Duration::from_secs(10));
+
+        // Read first
+        for client in &clients {
+            let start = Instant::now();
+            let _v = rt.block_on(client.get_ledger_information())?;
+            let duration = start.elapsed();
+            info!(
+                "Fetch from {:?} took {}ms",
+                client.path_prefix_string(),
+                duration.as_millis(),
+            );
+        }
```

Comment:
can we extract this code into a function so it is clear what it does?

Also serially calling all clients in this loop, can add a lot of latency (especially for larger 30-60 validator sets). let's make sure to issue the calls in parallel?